WebRoot/js/zimbraMail/share/view/ZmTagMenu.js

```diff
@@ -183,8 +183,27 @@ function(tagList, addRemove) {
 		var tagName = addRemove.add[i];
 		this._addNewTag(this, tagName, tagList, true, null, this._addHash);
 	}
+	
+	var removeList = addRemove.remove;
+	if (removeList.length > 0) {
+		for (i = 0; i < removeList.length; i++) {
+			var tagName = removeList[i];
+			var tagHtmlId = 'Remove_tag_' + i;
+			this._addNewTag(this, tagName, tagList, false, null, this._removeHash, tagHtmlId);
+		}
+		//remove all tags
+		new DwtMenuItem({parent:this, style:DwtMenuItem.SEPARATOR_STYLE});
+		var mi = new DwtMenuItem({parent:this, id:"REMOVE_ALL_TAGS", className: "ZAddNewMenuItem"});
```

Comment:
its a generic class name I guess Kalpesh added before which I am just using it to style the add items DwtMenuItem differently.